bin/i18n/sync-codeorg-in

```diff
@@ -7,57 +7,106 @@
 
 require File.expand_path('../../../pegasus/src/env', __FILE__)
 require 'fileutils'
+require 'json'
 require 'yaml'
 require 'tempfile'
 
+require_relative 'utils'
+
 def pull_in_files_for_translation
   localize_instructions
-  exec "bin/i18n-codeorg/in.sh"
+  run_bash_script "bin/i18n-codeorg/in.sh"
 end
 
-def find_instruction(f)
-  pattern = /^\s*"instructions": (".*?"),?\n$/
-  f.each_line do |line|
-    matches = line.match pattern
-    if matches
-      return matches.captures.first
-    end
-  end
-  nil
-end
-
-def copy_instructions_to_yml(instructions)
-  File.open("dashboard/config/locales/instructions.en.yml", "w+") do |f|
-    f.write(({ "en" => { "data" => { "instructions" => instructions }}}).to_yaml)
+def copy_to_yml(label, data)
+  File.open("dashboard/config/locales/#{label}.en.yml", "w+") do |f|
+    f.write(({ "en" => { "data" => { label => data }}}).to_yaml)
     File.write(f, File.read(f))
   end
 end
 
 def reformat_quotes
-  temp_file = Tempfile.new("tempinstructions.yml")
-  File.open("dashboard/config/locales/instructions.en.yml", "r") do |f|
-    f.each_line { |line| temp_file.puts line.gsub("'\"", '"').gsub("\"'", '"').gsub("''", "'")}
+  filenames = ["instructions", "markdown_instructions"]
+
+  filenames.each do |filename|
+    temp_file = Tempfile.new("temp#{filename}.yml")
+    File.open("dashboard/config/locales/#{filename}.en.yml", "r") do |f|
+      f.each_line { |line| temp_file.puts line.gsub("'\"", '"').gsub("\"'", '"').gsub("''", "'")}
+    end
+    temp_file.close
+    FileUtils.mv(temp_file.path, "dashboard/config/locales/#{filename}.en.yml")
   end
-  temp_file.close
-  FileUtils.mv(temp_file.path, "dashboard/config/locales/instructions.en.yml")
 end
 
 # Pull in instructions for levelbuilder levels from .level files
 # and save them to instructions.en.yml to be translated
```

Comment:
Want to update this comment and maybe method name from `localize_instructions` to `localize_level_content`?